spec/registration_server_spec.rb

```diff
@@ -9,11 +9,13 @@
   let(:port) { 9000 }
   let(:http_server) { double("http_server") }
   let(:certs_factory) { double("certs factory") }
+  let(:updates_url) { "https://updates.suse.com/sles12" }
```

Comment:
You're right: the HTTP thing is correct but looks confusing (this one is a unit test for RegistrationServer which does not matter at all if you're using HTTP or HTTPS).

On the other hand, I've fixed the URI thingie.